roles/system_packages/tasks/main.yml

```diff
@@ -46,9 +46,19 @@
   delay: "{{ retry_stagger | random + 3 }}"
   when: not (ansible_os_family in ["Flatcar", "Flatcar Container Linux by Kinvolk"] or is_fedora_coreos)
   loop:
+    - packages:
+        '*': [true]
+      state: latest
+      action_label: update
```

Comment:
Would it be possible to add a toggle variable for upgrading all packages? This way, the upgrade would only be enabled when necessary, and by default it could remain set to false.